### ISSUE TITLE
On some products, the Environment tab shows the contributor tab

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#3581]
![image](https://user-images.githubusercontent.com/1689815/98836733-535d5880-2442-11eb-94b1-119dfbf16650.png)
Report by @stephanegigandet 

- [ ] On some products, the Environment tab shows the contributor tab
- [ ] It would seem that it only happens on 1st navigation
- [ ] I can't reproduce, with or without contribution tab enabled

**Comment by VaiTon** [comment]
@teolemon are you able to reproduce on latest develop?

**Comment by mrudultora** [comment]
@VaiTon Yes. It is an active issue. This happens on opening the product very first time.

**Comment by mrudultora** [comment]
Unable to reproduce as of today on latest develop. Maybe fixed.

**Comment by VaiTon** [comment]
@teolemon are you able to reproduce?
